src/main/java/com/elastic/support/monitoring/MonitoringExportApp.java

```diff
@@ -17,7 +17,6 @@ public static void main(String[] args) {
             System.exit(0);
         }
 
-        MonitoringExportService service = new MonitoringExportService(monitoringExportInputs);
-        service.execExtract();
+        new MonitoringExportService().execExtract(monitoringExportInputs);
```

Comment:
Seems like this method makes more sense as a `static` one now that its only attribute is the logger?

Reply:
You could actually say that about a number of the current classes. But I don't want to do that yet - I still may change my mind.